specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/Settings.json

```diff
@@ -0,0 +1,594 @@
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SettingsName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Update EyesOn settings.": {
+            "$ref": "./examples/settings/UpdateEyesOnSetting.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Updates setting.",
+        "operationId": "ProductSettings_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/OperationalInsightsResourceProvider"
+          },
+          {
+            "$ref": "../../../common/1.0/types.json#/parameters/WorkspaceName"
+          },
+          {
+            "$ref": "#/parameters/SettingsName"
+          },
+          {
+            "$ref": "#/parameters/Settings"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Settings"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/1.0/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/settings/onboard": {
+      "post": {
+        "x-ms-examples": {
+          "Onboard workspace": {
+            "$ref": "./examples/settings/Onboard.json"
+          }
+        },
+        "tags": [
+          "Settings"
+        ],
+        "description": "Onboards workspace to sentinel.",
+        "operationId": "Workspace_Onboard",
```

Comment:
Operation Group (first section if OperationId) should be plural, and is used to group the settings.  Since workspace isn't really a resource, based on the Uri, this should likely be "ProductSettings_OnboardWorkspace"